tests/resources/expected_configs.json

```diff
@@ -88,6 +94,7 @@
       "NOT_NULL_SIMPLE",
       "REGEX_VALID_EMAIL",
       "CUSTOM_SQL_LENGTH_LE_30",
+      {"CUSTOM_SQL_LENGTH_LE_PARAMETRIZED": {"upper_bound": 0}},
```

Comment:
I might be misunderstanding this, but shouldn't the parameter AST be `{"upper_bound": 30}` to match `tests/resources/dq_rules_configs.yml`?

Reply:
+1

Reply:
I wanted to have a different test for 0 valued arguments. This resource is part of is a unit test. The `dq_rules_configs.yml` file is part of a integration test so there I wanted it to match the output of a non-parametrized rule.

Reply:
sounds good